public/styles.css

```diff
@@ -0,0 +1,832 @@
+  opacity: 0.3;
+}
+
+input:invalid {
+  border: rgb(229, 66, 66) 3px solid;
+}
+
+.input-error {
+  border: red 1px solid;
+}
+
+.list-group-item {
+  position: relative;
+  display: block;
+  padding: 0.5rem 1rem;
+  text-decoration: none;
+  color: black;
+}
+
+.mb-25 {
+  margin: 0.25rem 0rem;
+}
+
+.notification-small {
+  opacity: 0.8;
+}
+
+.justify-space-between {
+  justify-content: space-between;
+}
+
+.justify-space-evenly {
+  justify-content: space-evenly;
+}
+
+.navigation-bar {
+  background-color: #3060e4;
+  border-radius: 5px;
+}
+
+.navigation-list a {
+  list-style: none;
+  text-decoration: none;
+  color: white;
+  padding: 1rem;
+}
+.navigation-list a:hover {
+  background-color: #bfdbfe;
+  color: #023e8a;
+}
+
+.navigation-list li {
+  text-decoration: none;
+  color: white;
+  display: inline-block;
+  padding: 1rem 0rem;
+}
+
+.notifications {
+  border: 1px solid rgba(59, 112, 255, 0.5);
+  color: #3b70ff;
+  border-top: none;
+}
+
+.notifications:first-child {
+  border-top: 1px solid rgba(59, 112, 255, 0.5);
+}
+
+.example {
+  width: 25rem;
+}
+
+.navBar {
+  position: relative;
+}
+
+.navBar button {
+  position: fixed;
+  left: 40px;
+  top: 40px;
+  z-index: 10;
+  cursor: pointer;
+}
+
+.menuNav {
+  overflow: auto;
+  position: fixed;
+  top: 0;
+  right: 0;
+  background-color: white;
+  width: 0;
+  height: 100vh;
+  z-index: 99;
+  margin-top: 0rem;
+  transition: 0.5s;
+  border-top: 6px solid #023e8a;
+}
+
+.menuNav.showMenu {
+  width: 100%;
+}
+
+.hamMenu-items {
+  margin: 0rem 1.5rem;
+  margin-left: 3rem;
+  color: #023e8a;
+}
+
+.hamMenu-div {
+  padding: 0.5rem 0rem;
+  cursor: pointer;
+  width: 8rem;
+}
+
+.hamMenu-div:hover {
+  border-bottom: 4px solid #023e8a;
+  font-weight: 700;
+}
+
+.component-text {
+  color: #495057df;
+  line-height: 1.5rem;
+}
+
+.component > h1 {
+  font-weight: 800;
+}
+
+.component-box {
+  margin: 1rem auto;
+  padding: 0.5rem 0.5rem;
+  border-radius: 7px;
+  box-shadow: rgba(0, 0, 0, 0.12) 0px 0px 2.5px;
+}
+
+.component h3 {
+  font-weight: 600;
+}
+
+.component-sub {
+  margin: 2rem 0rem;
+}
+
+.component-iframe {
+  width: 100%;
+  height: auto;
+  border: 0;
+  border-radius: 10px;
+
+  background-color: #fafafa;
+}
+
+.modal {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  margin: auto;
+  overflow: auto;
+  z-index: 50;
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.modal-content {
+  width: 50%;
+  margin: 15rem auto;
+}
+
+.toast-content {
+  width: 50%;
+  position: fixed;
+  left: 25%;
+  right: 25%;
+  bottom: 10px;
+  animation: 1s;
+  z-index: 100;
+  display: none;
+}
+
+.inline {
+  display: inline-flex;
+}
+
+.showDisplay {
+  display: block;
+}
+
+.grid {
+  display: grid;
+  grid-gap: 1rem;
+}
+
+.grid-header {
+  display: grid;
+  grid-template-areas: "column-header column-header";
+  grid-template-columns: 1fr auto;
+}
+
+.column-header {
```

Comment:
Your css looks dope mann. You have tried to follow the atomic css approach as much as you can. Really impressive and learning for me as well :)